src/poliastro/twobody/events.py

```diff
@@ -25,8 +30,78 @@ def terminal(self):
     def last_t(self):
         return self._last_t * u.s
 
-    def __call__(self, t, u, k):
+    def __call__(self, t, u_, k):
```

Comment:
I noticed the `u` inside `__call__` collided with `astropy.units` which is also imported as `u`. Since `astropy.units` is generally imported as `u`, I changed this variable's name instead...

Reply:
Good catch @Yash-10!